Include/Acidanthera/Library/OcBootManagementLib.h

```diff
@@ -360,6 +386,11 @@ typedef struct OC_BOOT_CONTEXT_ {
 **/
 #define OC_SCAN_ALLOW_FS_EXT             BIT12
 
+/**
+  Allow scanning XBOOTLDR filesystems.
+**/
+#define OC_SCAN_ALLOW_FS_XBOOTLDR        BIT13
```

Comment:
Yes, it is BootLoaderSpec specific, i.e. it might barely be used anywhere by anyone. It means that a partition other then the ESP is marked (by its type GUID) as the partition on which to find (all) boot info. I believe it is intended to be a larger-sized FAT partition, in order to fit all the copies of all the kernel files there.

(It is not the same as the systemd approach used by Fedora, where the kernel files more sensibly stay on Linux's own boot partition.)